CONTRIBUTING.md

```diff
@@ -219,6 +219,15 @@ The commit message of your PR should be able to communicate what problem/opportu
 - Wrap the body at 72 characters
 - Use the body to explain what and why vs. how
 
+The commit message should be done with git and for that run the following command in your terminal:
+
+```sh
+$ git commit
```

Comment:
Can you change to `git commit <files changed>` ? I don't think just `git commit` will work.

Reply:
I guess that's for git add
Even for this changes I did the following:
```sh
$ git add <file name>
```

```sh
$ git commit
```

Reply:
Doh! @sayantani11 you are right :) Apologies, I was definitely thinking about git add when I made that comment. This looks good.

Reply:
Hahaha..... Happens @rnjudge